src/Rules/RuleLevelHelper.php

```diff
@@ -62,7 +62,7 @@ private function transformCommonType(Type $type): Type
 		}
 
 		return TypeTraverser::map($type, function (Type $type, callable $traverse) {
-			if ($type instanceof TemplateMixedType) {
+			if ($type instanceof TemplateMixedType && $this->checkExplicitMixed) {
```

Comment:
`TemplateMixedType` extends explicit `MixedType`, so I made its behavior consistent with that.

I wonder whether `@template T` and `@template T of mixed` should be implicit/explicit respectively. But I assume that the implicit mixed exists mainly for untyped legacy code. So in that sense it seems right to me for both of them to be considered explicit, even though it feels slightly inconsistent with how it works for non-generic parameters. Especially, considering that `checkImplicitMixed` is off even with level 9.